src/components/event-toggles/scss/module.scss

```diff
@@ -55,9 +56,11 @@ md-combobox {
   z-index: 2;
   margin: 0 0.5rem;
   box-shadow: 0 2px 4px rgba(0, 0, 0, 0.16), 0px 1px rgba(0, 0, 0, 0.18);
+  display: none;
 
   &.expanded {
     overflow: visible;
     transform: translateX(0);
+    display: inline-block;
```

Comment:
Do you still need the transform after changing display?

Reply:
probably not. Let me remove the transform line and try it out.